.github/ISSUE_TEMPLATE/devops_ticket.md

```diff
@@ -0,0 +1,24 @@
+---
+name: :boom: DevOps ticket
+about: Help us manage our deployed App.
+labels: devops
+title: :boom: [DevOps]
```

Comment:
hey @alina-beck ... I wasn't really able to copy the image here, do you know how I can do that?

Reply:
Not sure what you mean... I usually go to this list: https://gist.github.com/rxaviers/7360908, then select and copy the icon (`💥`) instead of the text (`:boom:`) – it's just a character!

Reply:
I was able to do this with the :wrench: that you added as a suggestion to my `PR`, but for some reason when I try to copy from the list you shared :point_up_2: it just copies `boom` not the icon, and without the `:`s ... most cases the `:`boom`:` works, but not for creating issue templates.. :worried: